app/controllers/api/users.php

```diff
@@ -28,6 +28,49 @@
 use Utopia\Validator\Range;
 use Utopia\Validator\Boolean;
 use MaxMind\Db\Reader;
+use Utopia\Validator\Integer;
+
+function createUser(string $hash, mixed $hashOptions, string $userId, string $email, string $password, string $name, Database $dbForProject, Stats $usage, Event $events): Document
```

Comment:
We should add a todo to remove this function when we move to using utopia/platform.